src/ncp/ncp_base_mtd.cpp

```diff
@@ -682,7 +682,7 @@ template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_THREAD_ASSISTING_PORT
     otError         error      = OT_ERROR_NONE;
 
     // First, we need to remove all of the current assisting ports.
-    for (; numEntries != 0; ports++, numEntries--)
+    for (; numEntries != 0; numEntries--)
```

Comment:
add comment to describe the reason for better readability? Easy to get confused on your proposal without following background.
`ports` points to the first element of internal port arrays, and each `otIp6RemoveUnsecurePort()` operation will cause following elements to be moved forward, so that `ports` always point to first valid element until all are removed.
It would be better to add note in the doxygen for  `otIp6RemoveUnsecurePort()` as well.
Thoughts?

Reply:
Updated.